### PR TITLE
Feature - training in profiles

### PR DESCRIPTION
**Pull request by joefitter**
* Added new training in profiles section encompassing exemptions
* Removed training from pil - replace with page playing back the training table and confirmation it is up to date
* Added redirect to session, provides link back to pil/project
* Added training modules to project init

### CODE REVIEW

**Comment by joefitter** [comment]
WIP until https://github.com/UKHomeOffice/asl-projects/pull/600

File: pages/pil/training/schema/index.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+  update: {
+    inputType: 'radioGroup',
+    format: toBoolean,
+    options: [
+      {
+        label: 'Yes, I need to update it',
+        value: true
+      },
+      {
+        label: 'No, that looks fine',
```
Comment (lennym) [inline pages/pil/training/schema/index.js:13]: image

File: pages/success/index.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ module.exports = () => {
 
   app.use((req, res, next) => {
     const taskId = get(req.session, 'success.taskId');
+    console.log('HERE', taskId);
```
Comment (lennym) [inline pages/success/index.js:101]:
```suggestion
```

File: pages/training/modules/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        const isExemption = get(req.session, `form[${req.model.id}].values.isExemption`);
+        req.form.schema = getSchema(isExemption);
+        next();
+      },
+      locals(req, res, next) {
+        res.locals.static.isExemption = get(req.session, `form[${req.model.id}].values.isExemption`);
+        next();
+      }
+    })(req, res, next);
+  });
```
Comment (lennym) [inline pages/training/modules/index.js:28]: This is nested inside itself in a weird way. Just do `app.use(form({...}))` directly.

File: pages/success/index.js (hunk 2 above)
Comment (joefitter) [reply pages/success/index.js:101]: lol

File: pages/pil/training/schema/index.js (hunk 1 above)
Comment (joefitter) [reply pages/pil/training/schema/index.js:13]: I know right

File: pages/profile/read/views/index.jsx [hunk 4]
```diff
@@ -50,6 +50,20 @@ const Index = ({
           <Profile profile={model} establishment={establishment} allowedActions={allowedActions} isOwnProfile={isOwnProfile} />
         </div>
       </div>
+      {
+        allowedActions.includes('training.read') && (
+          <section className="profile-section">
+            <h3><Snippet>training</Snippet></h3>
+            {
+              !!certificates.length && <TrainingSummary certificates={certificates} />
```
Comment (lennym) [inline pages/profile/read/views/index.jsx:58]: It feels like this check would be better done inside `<TrainingSummary />`

i.e.

```
if (!certificates || !certificates.length) {
  return null;
}
```

Comment (lennym) [reply pages/profile/read/views/index.jsx:58]: It would negate the need to default the certificates here as well.

Comment (joefitter) [reply pages/profile/read/views/index.jsx:58]: it already is, I just need to remove the check here

**Pull request merged**